src/main/java/org/sqlite/util/OSInfo.java

```diff
@@ -161,7 +160,7 @@ static String getHardwareName() {
         try {
             return processRunner.runAndWaitFor("uname -m");
         } catch (Throwable e) {
-            logger.atError().setCause(e).log("Error while running uname -m");
+            LogHolder.logger.atError().setCause(e).log("Error while running uname -m");
```

Comment:
i guess a problem would be that the log line would reference the wrong class name, since it is using a logger generated in another class.